src/Discovery/Discovery.tsx

```diff
@@ -293,6 +348,13 @@ const Discovery: React.FunctionComponent<DiscoveryBetaProps> = (props: Discovery
     <div className='discovery-container'>
       { (config.features.pageTitle && config.features.pageTitle.enabled)
       && <h1 className='discovery-page-title'>{config.features.pageTitle.text || 'Discovery'}</h1>}
+      <div className='g3-accessibility-links' id='discovery-page-accessibility-links'>
```

Comment:
Do we have a design for these accessibility links? I was hope it can have a label or something say "hey this is a group of accessibility links and you can use them to navigate to different compoments", because it has been a bit of confusing when I first saw them

Maybe an help icon with tooltip?

Reply:
oh and why do we only have accessibility links for Discovery and Explorer but not other pages?

Reply:
that's a great idea, i'll add a tooltip. and -- the reason is that the 508 standard only requires them in areas where there are blocks of repeated content so that a keyboard navigator can bypass those blocks. those two pages have the most repeated content to skip through, others don't have much below the primary area (profile page api keys, query page)

Reply:
there's no design, but i have a reference here: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html    the part where it says "On this page:"